packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -575,14 +565,17 @@ class MemoryBodyState extends State<MemoryBody>
   }
 
   Widget _buildMemoryControls(TextTheme textTheme) {
+    final androidChartButton = controller.isConnectedDeviceAndroid ||
+            controller.isOfflineAndAndroidData
+        ? createToggleAdbMemoryButton()
+        : const SizedBox();
+
     return Row(
       mainAxisSize: MainAxisSize.min,
       children: [
         _memorySourceDropdown(textTheme),
         const SizedBox(width: defaultSpacing),
-        controller.isConnectedDeviceAndroid
-            ? createToggleAdbMemoryButton()
-            : const SizedBox(),
+        androidChartButton,
```

Comment:
instead of making this a SizedBox when the app is not android, just do the if check directly in the build method
```dart
if (controller.isConnectedDeviceAndroid || controller.isOfflineAndAndroidData)
  createToggleAdbMemoryButton(),
```

Reply:
Done.